front/src/main/ui/pages/Home/ThirdPage/BigRoomBlock/BigRoomBlock.module.css

```diff
@@ -0,0 +1,36 @@
+.bigRoomBlock{
+    width: 630px;
+    height: 630px;
+    background: #C4C4C4;
+    border-radius: 10px;
+    position: relative;
+    margin-left: 70px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.bigRoomBlock span{
```

Comment:
better use only class. it takes more time to find nested span rather class and definitely will not affect children

Reply:
done